packages/blade/src/components/Form/Selector/SelectorTitle.tsx

```diff
@@ -1,24 +1,31 @@
+import Box from '~components/Box';
 import { BaseText } from '~components/Typography/BaseText';
 
 const SelectorTitle = ({
   children,
   isDisabled,
+  size,
 }: {
   children: React.ReactNode;
   isDisabled?: boolean;
+  size: 'small' | 'medium';
 }): React.ReactElement => {
+  const isSmall = size === 'small';
   return (
-    <BaseText
-      lineHeight="l"
-      fontFamily="text"
-      fontWeight="regular"
-      color={
-        isDisabled ? 'surface.text.placeholder.lowContrast' : 'surface.text.subtle.lowContrast'
-      }
-      fontSize={100}
-    >
-      {children}
-    </BaseText>
+    <>
+      <Box marginLeft="spacing.2" />
+      <BaseText
+        lineHeight={isSmall ? 's' : 'l'}
```

Comment:
We're also going to have `large` size now. Better to maintain a size to lineHeight, fontSize mapping here?

Reply:
Done.